keyboards/handwired/ziyoulang_k3_mod/keymaps/default/keymap.c

```diff
@@ -0,0 +1,45 @@
+     * └────┴────┴────┴────────────────────────┴───┴───┴───┘└───┴───┴───┘└───┴───┴───┘
+     */
+    [0] = LAYOUT_96_ansi(
+        KC_ESC,           KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12, KC_PSCR,  KC_DEL,  KC_HOME, KC_INS,  KC_PGUP, KC_PGDN,
+
+        KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,                            KC_NUM,  KC_PSLS, KC_PAST, KC_PMNS,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,                            KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_CAPS,          KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,                             KC_P4,   KC_P5,   KC_P6,
+        KC_LSFT,          KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,          KC_RSFT,                            KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+                                                                                                                                               KC_UP,
+        KC_LCTL,          KC_LGUI, KC_LALT,                   KC_SPC,                             KC_RALT, MO(1),            KC_RCTL,                            KC_P0,            KC_PDOT,
+                                                                                                                                      KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_96_ansi(
+        QK_BOOT,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_END,  _______, KC_SCRL, KC_PAUS,
+
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                            _______, _______, _______, _______,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                            _______, _______, _______, _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                            _______, _______, _______,
+        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                            _______, _______, _______, _______,
+                                                                                                                                               _______,
+        _______,          _______, _______,                   _______,                            _______, MO(1),            _______,                            _______,          _______,
```

Comment:
This won't have worked.
```suggestion
        _______,          _______, _______,                   _______,                            _______, _______,            _______,                            _______,          _______,
```